### PR TITLE
LPD-35563 - Migrate AccessibilityMenuSmoke#BrowserLocalStorageSmoke Poshi test to Playwright

### PR DESCRIPTION
**Pull request by thiago-buarqque**
https://liferay.atlassian.net/browse/LPD-35563

**Flakyness**

I ran the test 50 times in a row to make sure it isn't flaky

### CODE REVIEW

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:playwright

**Comment by ethib137** [comment]
ci:report:32634393

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:playwright

**Comment by thiago-buarqque** [comment]
@ethib137 CI is failing due to other issues. The added test is passing. Let me know if you have any question regarding the changes I've made

File: modules/test/playwright/tests/accessibility-menu-web/accessibilityMenu.spec.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+				accessibilityMenuPage.underlinedLinksToggle
+			).toBeChecked();
+		});
+
+		await test.step('Verify that the "underlined links" toggle is on when logged out, then turn it off', async () => {
+			await performLogout(page);
+
+			await accessibilityMenuPage.openAccessibilityMenu();
+
+			await accessibilityMenuPage.toggleUnderlinedLinks(false);
```
Comment (ethib137) [inline modules/test/playwright/tests/accessibility-menu-web/accessibilityMenu.spec.ts:74]: Based on the test.step description we need to add an assertion that the toggle is on before toggling it off.

File: modules/test/playwright/tests/accessibility-menu-web/accessibilityMenu.spec.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+			await accessibilityMenuPage.openAccessibilityMenu();
+
+			await expect(
+				accessibilityMenuPage.underlinedLinksToggle
+			).toBeChecked();
+
+			await expect(
+				accessibilityMenuPage.underlinedLinksToggle
+			).toBeChecked();
```
Comment (ethib137) [inline modules/test/playwright/tests/accessibility-menu-web/accessibilityMenu.spec.ts:112]: Looks like an accidental duplicate of this assertion. Please remove one.

**Review by ethib137 — CHANGES_REQUESTED** [review]
Thanks @thiago-buarqque ... just a couple small changes. Thanks.

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:playwright

File: modules/test/playwright/tests/accessibility-menu-web/accessibilityMenu.spec.ts (hunk 2 above)
Comment (thiago-buarqque) [reply modules/test/playwright/tests/accessibility-menu-web/accessibilityMenu.spec.ts]: Ops, thanks

**Comment by thiago-buarqque** [comment]
@ethib137 Test is passing on CI, failures are unrelated. Please let me know if you have any other comments.

**Comment by ethib137** [comment]
LGTM! Thanks.

**Comment by ethib137** [comment]
ci:forward

**Pull request closed without merging**